app/src/main/java/com/sopt/dive/presentation/home/component/GiftButton.kt

```diff
@@ -0,0 +1,53 @@
+package com.sopt.dive.presentation.home.component
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.sopt.dive.R
+import com.sopt.dive.core.extention.noRippleClickable
+
+@Composable
+fun GiftButton(onClick: () -> Unit) {
+    val shape = RoundedCornerShape(20.dp)
+    val contentColor = Color.Red
+
+    Box(
+        modifier = Modifier
+            .noRippleClickable(onClick = onClick)
+            .border(BorderStroke(1.dp, contentColor), shape)
+            .clip(shape)
+            .padding(horizontal = 10.dp, vertical = 4.dp),
+        contentAlignment = Alignment.Center
+    ) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
```

Comment:
```suggestion
    Row(
        modifier = Modifier
            .noRippleClickable(onClick = onClick)
            .border(BorderStroke(1.dp, contentColor), shape)
            .clip(shape)
            .padding(horizontal = 10.dp, vertical = 4.dp),
        verticalAlignment = Alignment.CenterVertically
    ) {
```
이렇게 작성하면 불필요한 레이아웃 계층을 줄여 성능이 개선될 것 같아요 ! Box를 사용하신 구체적인 이유가 없다면 이렇게 바꾸는건 어떨까요 ??